example-configs/templates.conf

```diff
@@ -339,10 +339,10 @@ GIT_URL_template_kali = $(GIT_BASEURL)/fepitre/qubes-template-kali.git
 ################################################################################
 # Qubes-builder deps
 DEPENDENCIES ?=
-DEPENDENCIES += git rpmdevtools rpm-build createrepo perl-Digest-MD5 perl-Digest-SHA
+DEPENDENCIES += git rpmdevtools rpm-build perl-Digest-MD5 perl-Digest-SHA systemd-container
 
 # Additional for debian template
-DEPENDENCIES += debootstrap dpkg-dev
+DEPENDENCIES += debootstrap dpkg-dev createrepo
```

Comment:
createrepo isn't really for debian template, it should stay in the earlier section (you can wrap long line with `\` and the line end).

Reply:
Ah okay, I thought this is needed as seen [here](https://github.com/QubesOS/qubes-builder/blob/3ccf14b63dbff78576c549c693c58d98de0e8af7/Makefile#L55), but I guess this is a different context?

Reply:
Yup, that one is for building rpm packages on Debian.